src/layer/gutter.js

```diff
@@ -21,6 +21,24 @@ class Gutter{
 
         this.$lines = new Lines(this.element);
         this.$lines.$offsetCoefficient = 1;
+
+        this.spinnerRow = -1;
```

Comment:
could this be `null` instead of `-1`? it indicates "nothingness" better, and in case ace ever displays negative row numbers it would not cause any collisions

Reply:
yeah makes sense, fixed